action-sonos.py

```diff
@@ -50,45 +52,26 @@ def addSong_callback(hermes, intentMessage):
 def getInfos_callback(hermes, intentMessage):
     raise NotImplementedError("getInfos_callback() not implemented")
 
+def radioOn_callback(hermes, intentMessage):
+    raise NotImplementedError("radioOn_callback() not implemented")
 
-def playAlbum_callback(hermes, intentMessage):
-    use_case = PlayAlbumUseCase(hermes.device_discovery_service, hermes.music_search_service,
-                                hermes.music_playback_service)
-    play_album_request = PlayAlbumRequestAdapter.from_intent_message(intentMessage)
-
-    logging.info(play_album_request)
-    response = use_case.execute(play_album_request)
 
-    if not response:
-        logging.info(response)
-        hermes.publish_end_session(intentMessage.session_id, response.message)
-    else:
-        logging.info(response)
-        hermes.publish_end_session(intentMessage.session_id, response.feedback)
+def previousSong_callback(hermes, intentMessage):
+    raise NotImplementedError("previousSong_callback() not implemented")
 
 
-def playPlaylist_callback(hermes, intentMessage):
-    use_case = PlayPlaylistUseCase(hermes.device_discovery_service, hermes.music_search_service,
-                                   hermes.music_playback_service)
-    play_playlist_request = PlayPlaylistRequestAdapter.from_intent_message(intentMessage)
+def nextSong_callback(hermes, intentMessage):
+    usecase = NextTrackUseCase(hermes.device_discovery_service, hermes.device_transport_control_service)
 
-    logging.info(play_playlist_request)
-    response = use_case.execute(play_playlist_request)
+    next_track_request = NextTrackRequestAdapter.from_intent_message(intentMessage)
 
+    response = usecase.execute(next_track_request)
     if not response:
-        logging.info(response)
-        hermes.publish_end_session(intentMessage.session_id, response.message)
+        logging.debug(response.value)
+        hermes.publish_end_session(intentMessage.session_id, FR_TTS_SHORT_ERROR)
```

Comment:
File `feedback.py` could have a nicer structure so you could get the TTS using a dict t do something like `tts_dict['short_error']['fr']`. I think will make things more clear in the future. 
Maybe we should open an issue instead of dealing with it in this PR.

Reply:
Yes, I'll refactor this in a dict !